io_scene_nif/modules/object/object_export.py

```diff
@@ -187,7 +184,7 @@ def export_node(self, b_obj, space, parent_block, node_name):
             # -> mesh data.
             # If this has children or animations or more than one material it gets wrapped in a purpose made NiNode.
             is_collision = b_obj.game.use_collision_bounds
-            has_ipo = b_obj_ipo and len(b_obj_ipo.getCurves()) > 0
+            has_ipo = b_obj_ipo and b_obj_ipo.fcurves
```

Comment:
We should probably rename this variable as ipos are not a thing in blender 2.5+

Reply:
Fixed with latest commit, renamed to has_anim